### PR TITLE
bpo-29778: test_embed tests the path configuration

### PR DESCRIPTION
**Pull request by vstinner**
https://bugs.python.org/issue29778

### CODE REVIEW

**Comment by vstinner** [comment]
cc @zooba

File: Lib/test/test_embed.py [hunk 1]
```diff
@@ -471,6 +471,24 @@ class InitConfigTests(EmbeddingTestsMixin, unittest.TestCase):
...
+    COPY_PATH_CONFIG = [
+        # Copy core config to global config for expected values
+        #'program_full_path',
+        'prefix',
+        'exec_prefix',
+        #'module_search_path',
+        'program_name',
+        'home',
+    ]
+    if MS_WINDOWS:
```
Comment (zooba) [inline Lib/test/test_embed.py:494]: IIRC, base_executable applies on macOS too.

Comment (zooba) [reply Lib/test/test_embed.py:494]: But maybe only the sys module attribute is used and not the path config struct?

**Review by zooba — APPROVED** [review]

File: Lib/test/test_embed.py (hunk 1 above)
Comment (vstinner) [reply Lib/test/test_embed.py:494]: The idea of this PR is to test _PyPathConfig content: the global _Py_path_config variable.

sys.base_executable is set from PyConfig.base_executable which is already tested in test_embed.

**Comment by vstinner** [comment]
>  continuous-integration/travis-ci/pr Pending — The Travis CI build is in progress 

Stupid bot. https://travis-ci.org/github/python/cpython/builds/705808750 completed successfully 2 hours ago :-(

**Comment by vstinner** [comment]
> Stupid bot. https://travis-ci.org/github/python/cpython/builds/705808750 completed successfully 2 hours ago :-(

I restarted the whole Travis CI job...

**Pull request merged**